GAME/DELTAPAK.C

```diff
@@ -1136,15 +1142,43 @@ int Load_and_Init_Cutseq(int num)
 	return 0;
 }
 
-struct ITEM_INFO *cutseq_restore_item(int num)
+struct ITEM_INFO *cutseq_restore_item(int num)//2D738(<), 2DA20(<)
 {
-	S_Warn("[*cutseq_restore_item] - Unimplemented!\\n");
+	if(level_items > 0)
+	{
+		for(int i = 0; i < numnailed; i++)
+		{
+			if(items[i].object_number == num)
+			{
+				items[i].status = old_status_flags[numnailed];
+				items[i].flags = old_status_flags2[numnailed];
+				numnailed++;
+
+				return items + i;
+			}
+		}
+	}
+
 	return NULL;
 }
 
-void cutseq_kill_item(int num)
+void cutseq_kill_item(int num)//2D69C(<), 2D984(<) (F)
 {
-	S_Warn("[cutseq_kill_item] - Unimplemented!\\n");
+	if(level_items > 0)
+	{
+		for (int i = 0; i < level_items; i++) {
```

Comment:
Needs to be pure C or won't compile for PS1.

int i; should be declared at top of method.